cloudquery-cloud/README.md

```diff
@@ -0,0 +1,38 @@
+# CloudQuery Cloud
+
+![datadog-integration][1]
```

Comment:
```suggestion
```
We display a media carousel at the top of integration tiles. This is not needed.